pywps/inout/literaltypes.py

```diff
@@ -341,7 +335,9 @@ def make_allowedvalues(allowed_values):
 
     for value in allowed_values:
 
-        if isinstance(value, (AllowedValue, AnyValue, NoValue, ValuesReference)):
+        if value in ALLOWED_VALUES_TYPES:
+            new_allowedvalues.append(value())
+        elif isinstance(value, ALLOWED_VALUES_TYPES):
```

Comment:
should this be `isinstance(value, ALLOWED_VALUE_TYPE)`?

Reply:
Do you mean changing the name `ALLOWED_VALUES_TYPES` to `ALLOWED_VALUE_TYPE`?

Reply:
If not, the 2 checks are necessary, (`value in ALLOWED_VALUES_TYPES` and `isinstance(value, ALLOWED_VALUES_TYPES)`) because the defaut value for `LiteralInput.allowed_values` is `AnyValue` which is the class directly, instead of an instance.

Reply:
but `ALLOWED_VALUES_TYPES` it defined like this:
```
ALLOWED_VALUES_TYPES = (AllowedValue, AnyValue, NoValue, ValuesReference)
```

while there is also something like this:
```
from pywps.validator.allowed_value import ALLOWEDVALUETYPE
```

Reply:
Ah, I see... maybe I could rename ALLOWED_VALUES_TYPES to ALLOWED_VALUES_CLASSES? or maybe reference the classes by their specific names directly instead of creating this tuple...

It doesn't serve the same purpose, ALLOWEDVALUETYPE is just a container for the constants 'value' and 'range'.

Reply:
... I didn't know that  `isinstance` can also check against a tuple of classes. So, you can keep it like this ... maybe just a comment to avoid the confusion.